modules/pel/peltool/peltool.py

```diff
@@ -427,9 +459,19 @@ def main():
         else:
             if not os.path.isdir(args.list):
                 sys.exit(f"{args.list} is not a valid directory")
+            if args.pelID:
+                parsePelFromID(args.list, args.pelID, config)
+                sys.exit(0)
             listOption(args.list, config, args.extension)
         sys.exit(0)
 
+    if args.pelID:
+        if inBMC:
+            parsePelFromID(BMCPELsPath, args.pelID, config)
+        else:
+            sys.exit("Use -i/--id along with -l")
```

Comment:
Could the error message be more descriptive?

`To obtain specific PEL details using the PEL tool outside of the BMC environment, please provide the path to the PELs using the -i/--id option in conjunction with -l.`?

If you felt its too lengthy then please rephrase it,